ksiren-moshi-adapter/src/main/java/com/brightspace/ksiren/moshi_adapter/KSirenMoshiWriter.kt

```diff
@@ -11,22 +11,22 @@ class KSirenMoshiWriter(
 		writer.serializeNulls = false
 	}
 
-	override fun beginObject() : KSirenJsonWriter{
+	override fun beginObject(): KSirenJsonWriter {
```

Comment:
Hope you used automatic `Reformat Code` action for this.

![image](https://user-images.githubusercontent.com/2973055/90798769-5a137e00-e2e0-11ea-9523-de117ef7c30e.png)

Reply:
Yes, it's why there are some unrelated whitespace changes in files where code was changed: https://github.com/Brightspace/ksiren/pull/11#discussion_r470892351